packages/frontend/src/pages/Others/ManagementReports/index.tsx

```diff
@@ -1,3 +1,48 @@
+import { Button, TextField } from '@mui/material';
+import { useState } from 'react';
+import { ContainerWrapper } from './index.styles';
+import FileUploadService from '../../../services/FileUploadService';
+
+const Form = () => {
+  const selectFile = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { files } = event.target;
+    const selectedFiles = files as FileList;
+    setFile(selectedFiles?.[0]);
+  };
+  const upload = () => {
+    if (!file) return;
+
+    FileUploadService.upload(nodeId, file).catch((err) => {
+      if (err.response && err.response.data && err.response.data.message) {
+      } else {
+        console.log('something went wrong!');
+      }
+
+      setFile(undefined);
+    });
+  };
+  const [file, setFile] = useState<File>();
+  const [nodeId, setNodeId] = useState<string>('');
+  return (
+    <>
+      <TextField type="file" onChange={selectFile} />
```

Comment:
Should file upload accept only a few file formats (PDF, Excel, Image, PowerPoint, Word, PlainText)? You could find `mimeTypesMapping` in file luceneQueryBuilder.ts

e.g. `<TextField type="file" inputProps={{ accept:"application/pdf" }} onChange={selectFile} />`

Reply:
Removed this frontend code since it'll not be used in production anyway. We can discuss about this in the future.